README.md

```diff
@@ -20,7 +20,7 @@ dependencies {
 In the following cases, CircleCI will publish a new version with the following format to our remote Maven repo:
 
 * For each commit in an open PR: `<PR-number>-<commit full SHA1>`
-* Each time a PR is merged to `develop`: `develop-<commit full SHA1>`
+* Each time a PR is merged to `trunk`: `trunk-<commit full SHA1>`
```

Comment:
👍 
I just checked on Buildkite (it case we forgot to configure it to do this for trunk too or explicitly listed branch names on which to publish or to exclude, etc)… and I confirmed that, just as expected, commits to `trunk` also publishes to s3 with such version name, so all good there 👍